resident/resident-service/src/main/java/io/mosip/resident/constant/TransactionStage.java

```diff
@@ -33,7 +33,11 @@ public static Optional<String> getTypeCode(String transactionTypeCode, Environme
 	}
 
 	private static List<String> getTransactionTypeCodeList(TransactionStage transactionStage, Environment env) {
-		return List.of(env.getProperty(transactionStage.transactionTypeCodePropertyName).split(","));
+		List<String> transactionTypeCode = null;
```

Comment:
Initialize it with empty list. (List.of())

Reply:
Updated as per review comment